### ISSUE TITLE
planemo serve - automatically upload test-data

### ISSUE THREAD

**Issue by scholtalbers** [galaxyproject/planemo#875]
Note: It may well be that I just don't find the right argument?

It would be nice for `planemo serve` to automatically upload the files in `test-data` to the current history upon startup as I find myself uploading test-data quite often (although less so due to the dynamic reload!).

**Comment by mvdbeek** [comment]
Agree, that would be nice, but did you know that they are preloaded in the ftp folder ?
